cycledash/templates/runs.html

```diff
@@ -77,9 +78,16 @@
           {%- if run['tumor_bam_uri'] %}<span class="label label-info" title="Has an associated tumor BAM">tumor</span>{% endif -%}
           {%- if run['normal_bam_uri'] %}<span class="label label-info" title="Has an associated normal BAM">normal</span>{% endif -%}
         </td>
+        <td class="comments">
+          <span class="comment-bubble {{ '' if run['num_comments'] > 0 else 'no-comment-dimmed'}}">
```

Comment:
You don't need the `> 0` since 0 is falsey… you may know this and chose to do it anyway.  Just FYI.